### ISSUE TITLE
(Fixed) Quotes written to in Settings / Default ACLs / Advanced ACLs get converted to &quote

### ISSUE THREAD

**Issue by ra-at-diladele-com** [diladele/websafety#481]
Steps to reproduce:
1. navigate to UI / Squid / Settings / Default ACLs / Advanced
2. Type the following ```acl temp_user src "/etc/squid/sq.ip_temp_user"```
3. Save and Restart

There will be ```acl temp_user src &quot;/etc/squid/sq.ip_temp_user&quot;``` in access_controls.conf instead of expected value ```acl temp_user src "/etc/squid/sq.ip_temp_user"```

**Comment by ra-at-diladele-com** [comment]
Workaround - same string written to UI / Squid / ICAP / Exclusions / Advanced gets into advanced.conf correctly...

**Comment by ra-at-diladele-com** [comment]
Workaround - in /opt/websafety/var/console/squid/templates/squid/conf/access_controls.conf at line 540 and several lines below you need to wrap it with ```{% autoescape off %}``` and ```{% endautoescape %}``` django template directives  so that it looks like:

```
# advanced acl written as-is without autoescape
{% autoescape off %}
{% if acldefault.advanced_acl %}{{acldefault.advanced_acl}}
{% else %}# no advanced acls specified
{% endif %}
{% endautoescape %}
```

**Comment by ra-at-diladele-com** [comment]
Fixed need to add it to known issues with version 5.2 to docs.diladele.com. Fixed in 6.0+.
